packages/functional-tests/tests/subscription-tests/coupons-ui-tests/coupon.spec.ts

```diff
@@ -141,7 +141,9 @@ test.describe('coupon test', () => {
     expect(await relier.isPro()).toBe(true);
   });
 
-  test('subscribe with paypal and use coupon', async ({
+  //Diabling the test as this is being flaky because Paypal Sandbox is being finicky
```

Comment:
nit: typo on "Disabling" (here and below), but not worth the time/energy to fix and rerun CI.
Merge ahoy!